src/ProgramBlockEditor.scss

```diff
@@ -73,56 +73,54 @@
     }
 }
 
-button.btn.ProgramBlockEditor__program-deleteAll-button {
+.ProgramBlockEditor__program-deleteAll-button {
     background-color: #4C9990;
     border: none;
     border-radius: 50%;
     box-shadow: none;
     margin-left: 1.5rem;
-    padding: 0.5rem;
 
     &:focus {
         box-shadow: 0 0 0 $c2lc-focus-indicator-width black;
     }
 }
 
-button.btn.ProgramBlockEditor__program-deleteAll-button--pressed {
+.ProgramBlockEditor__program-deleteAll-button--pressed {
     background-color: #000;
 }
 
-button.btn.ProgramBlockEditor__program-deleteAll-button.ProgramBlockEditor__program-deleteAll-button--disabled,
-button.btn.ProgramBlockEditor__program-deleteAll-button.ProgramBlockEditor__program-deleteAll-button--disabled:hover,
-button.btn.ProgramBlockEditor__program-deleteAll-button.ProgramBlockEditor__program-deleteAll-button--disabled:active {
+.ProgramBlockEditor__program-deleteAll-button.ProgramBlockEditor__program-deleteAll-button--disabled,
```

Comment:
We can remove all specific state styles to the deleteAllButton, as we want it to be consistent with other IconButtons.

Reply:
Keep .ProgramBlockEditor__program-deleteAll-button-svg style, and let'e remove all other styles.